adding size comparison

abdelaziz-mahdy opened the pull request.

abdelaziz-mahdy commented:
this is an example svg that can be generated 
<img width="419" alt="image" src="https://github.com/user-attachments/assets/7dbe72d6-ae3c-49c2-acaf-93d38d8d1ce9">
if you want to edit the readme using the script instead of the svg let me know

rainyl commented:
> this is an example svg that can be generated <img alt="image" width="419" src="https://private-user-images.githubusercontent.com/25157308/380554481-7dbe72d6-ae3c-49c2-acaf-93d38d8d1ce9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAwODMzNjAsIm5iZiI6MTczMDA4MzA2MCwicGF0aCI6Ii8yNTE1NzMwOC8zODA1NTQ0ODEtN2RiZTcyZDYtYWUzYy00OWMyLWFjYWYtOTNkMzhkOGQxY2U5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI4VDAyMzc0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg5MjJiMzcyZTg5MDM3Zjk0ZjM2YzU4ODNkYzNjY2U1ZjRjYjRiYjczMWQ0Yzk1NmZhYmJhNjFhMzkwNTYxMWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.f97gEeCKtS8Akl7bwsgCUd5UwOQNYQX90PXTrYoaIKE"> 

Wow, looks soooo great! 🎉  

> if you want to edit the readme using the script instead of the svg let me know

maybe we can save the generated svg to `images/` directory and add a section in README to show it?

abdelaziz-mahdy commented:
> > this is an example svg that can be generated <img alt="image" width="419" src="https://private-user-images.githubusercontent.com/25157308/380554481-7dbe72d6-ae3c-49c2-acaf-93d38d8d1ce9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAwODMzNjAsIm5iZiI6MTczMDA4MzA2MCwicGF0aCI6Ii8yNTE1NzMwOC8zODA1NTQ0ODEtN2RiZTcyZDYtYWUzYy00OWMyLWFjYWYtOTNkMzhkOGQxY2U5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI4VDAyMzc0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg5MjJiMzcyZTg5MDM3Zjk0ZjM2YzU4ODNkYzNjY2U1ZjRjYjRiYjczMWQ0Yzk1NmZhYmJhNjFhMzkwNTYxMWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.f97gEeCKtS8Akl7bwsgCUd5UwOQNYQX90PXTrYoaIKE"> 
> 
> Wow, looks soooo great! 🎉  
> 
> > if you want to edit the readme using the script instead of the svg let me know
> 
> maybe we can save the generated svg to `images/` directory and add a section in README to show it?

That will work yes, and we can show each package svg in it's readme

rainyl commented:
> That will work yes, and we can show each package svg in it's readme

Cool!

abdelaziz-mahdy commented:
Will do it tomorrow, since it's late for me😅

abdelaziz-mahdy commented:
i think its correct but the commit is failing

abdelaziz-mahdy commented:
i added full apk size comparison too, and for the commit problem i made the workflow work on my branch and it works correctly, also i added them to the readme 

let me know if there is anything that needs to be added or updated

rainyl commented:
> i added full apk size comparison too, and for the commit problem i made the workflow work on my branch and it works correctly, also i added them to the readme
> 
> let me know if there is anything that needs to be added or updated

Great! I extracted the python scripts to reduce the length of the workflow.

Looks good to me now.

abdelaziz-mahdy commented:
> > i added full apk size comparison too, and for the commit problem i made the workflow work on my branch and it works correctly, also i added them to the readme
> > 
> > let me know if there is anything that needs to be added or updated
> 
> Great! I extracted the python scripts to reduce the length of the workflow.
> 
> Looks good to me now.

I wanted to do it but was not finding a good location for them 😅😅

Glad you did it.

rainyl commented:
Merged, thanks for your efforts! 🎉 @abdelaziz-mahdy
